community/kernel/src/main/java/org/neo4j/kernel/IdType.java

```diff
@@ -19,49 +19,83 @@
  */
 package org.neo4j.kernel;
 
-import org.neo4j.unsafe.impl.internal.dragons.FeatureToggles;
+import java.util.Arrays;
+import java.util.List;
 
 /**
  * @deprecated This will be moved to internal packages in the next major release.
  */
 @Deprecated
-public enum IdType
+public class IdType
```

Comment:
Hmm, now I don't get it actually. I though that you could have different settings for different dbs, but it still seems to be per JVM, i.e. that IdType is sort of like an enum. So what was this refactoring useful for?